apps/web/src/components/organisms/Layout/RootLayout.tsx

```diff
@@ -59,6 +60,7 @@ export function RootLayout({ children, backLink, heading = SERVICE_NAME, user }:
 
   return (
     <div className={`${primaryFont.variable} font-sans`}>
+      <CookieBanner />
```

Comment:
minor one but if the`<CookieBanner />` is at the bottom of the page then move this to the bottom of the layout for clarity

Reply:
its at the top of the html for accessibility, it should be one of the first element tabbed to for the user to be able to accept/deny cookies.